torchvision/prototype/transforms/_container.py

```diff
@@ -33,6 +33,7 @@ def extra_repr(self) -> str:
 
 
 class RandomChoice(Transform):
+    # datumbox: Shouldn't this be `transforms: Sequence[Callable]` for BC, similar to what we did for `Compose`?
     def __init__(self, *transforms: Transform, probabilities: Optional[List[float]] = None) -> None:
```

Comment:
here also arg name has changed `probabilities` vs `p`
https://github.com/pytorch/vision/blob/020eafe12a5c7f7e34972887e00007510927684b/torchvision/transforms/transforms.py#L556-L559

Reply:
Good spot. The new API is supposed to be as BC as possible. If we rename something, we need to go through a deprecation process and have the mechanism in place.

Reply:
https://github.com/pytorch/vision/pull/6498